scrapenews/spiders/rekordnorth.py

```diff
@@ -46,7 +49,7 @@ def parse_item(self, response):
 
             publication_date_str = response.xpath('//time/@datetime').extract_first()
             # u'2018-06-14T11:00:00+00:00'
-            publication_date = datetime.strptime(publication_date_str[0:19], '%Y-%m-%dT%H:%M:%S')
+            publication_date = lib.parse_date_hour_min_sec(publication_date_str)
```

Comment:
right, so there are three? different dates? I'm beginning to think it doesn't make sense to factor this out because it's actually quite specific code and wasn't hurting anybody? There are so many ways of writing date_hour?_min?_sec? - when I read this code I can immediately see what it's doing. When it's factored out, I need to go see which permutation of possible date formats this is.